graphql-proxy/src/utils/toCamelCase.ts

```diff
@@ -0,0 +1,9 @@
+export default (snakecase: string) => {
```

Comment:
What about adding some comments explaining what this function does? A couple of before-after examples would save a lot of brainpower.